providers/src/airflow/providers/common/sql/provider.yaml

```diff
@@ -25,6 +25,7 @@ state: ready
 source-date-epoch: 1730012422
 # note that those versions are maintained by release manager - do not update them manually
 versions:
+  - 1.19.1
```

Comment:
Please don't set versions unless explicitly asked by release manager.
It causes troubles during release

Reply:
Will revert that @eladkal had the same remark from @potiuk my bad

Reply:
I did it via Github UI / suggestions :)